gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -481,6 +481,7 @@
     [100013, 'current_version', 'VARCHAR', [], "UtilityFunctions::current_version"],
     [100014, 'last_query_id', 'VARCHAR', [], "UtilityFunctions::last_query_id"],
     [100015, 'uuid', 'VARCHAR', [], "UtilityFunctions::uuid"],
+    [100016, 'uniq_id', 'LARGEINT', [], "UtilityFunctions::uniq_id"],
```

Comment:
why not override the old uuid implementation

Reply:
Inconsistency return types may introduce some rolling upgrade issues.

Reply:
how about using the name uuid_integer() or uuid_numeric()?

Reply:
good idea